roles/bootstrap-os/tasks/debian.yml

```diff
@@ -62,3 +62,14 @@
     - '"changed its" in bootstrap_update_apt_result.stdout'
     - '"value from" in bootstrap_update_apt_result.stdout'
   ignore_errors: true
+
```

Comment:
HI @tu1h 

It the code in `ubuntu.yml` more resonable ?

Reply:
ubuntu.yml is a symlink to debian.yml, you mean we should make ubuntu.yml as the regular text file?

Reply:
Got it